3/week3/src/components/Buttons/Button.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import styles from "./Button.module.css";
+
+function Button({ buttonFunction, text }) {
+  return (
+    <button className={styles.btnAddMinus} onClick={buttonFunction}>
+      {text}
+    </button>
+  );
+}
+
+export default Button;
```

Comment:
It would probably make sense to memoize this. We don't want it re-rendering unnecesarily.

```suggestion
export default memo(Button);
```